src/helm.exec.ts

```diff
@@ -275,7 +275,7 @@ export function helmGet(resourceNode?: ClusterExplorerNode) {
     if (!resourceNode) {
         return;
     }
-    if ((resourceNode.nodeType !== NODE_TYPES.helm.release || NODE_TYPES.helm.history)) {
+    if (resourceNode.nodeType !== (NODE_TYPES.helm.history ||NODE_TYPES.helm.release)) {
```

Comment:
We still have a problem here because it will first check the result inside the brackets `(NODE_TYPES.helm.history || NODE_TYPES.helm.release)` and it will return a value based on the OR expression. 

The OR will return the first value of the expression if its value is true and because NODE_TYPES.helm.history will be always converted to true because it is always defined, it's like you're just doing `resourceNode.nodeType !== NODE_TYPES.helm.history`. If you try it you will see that it now works for heml.history nodes but it fails for heml.release nodes.

I'd suggest to simply convert it to `resourceNode.nodeType !== NODE_TYPES.helm.history && resourceNode.nodeType !== NODE_TYPES.helm.release`

Reply:
Sorry my bad I missed this. Fixing this condition